packages/front-end/pages/settings/index.tsx

```diff
@@ -760,7 +763,32 @@ const GeneralSettingsPage = (): React.ReactElement => {
                         })}
                       />
                     </div>
-
+                    <div className="mb-3  form-inline flex-column align-items-start">
+                      <SelectField
+                        label={"Multiple comparisons correction to use: "}
+                        className="ml-2"
+                        value={form.watch("pValueCorrection") ?? "none"}
+                        onChange={(value) => {
+                          const correction = value as PValueCorrection;
+                          form.setValue("pValueCorrection", correction);
```

Comment:
you can just directly do
`form.setValue("pValueCorrection", value as PValueCorrection);`